src/components/PrintModal/PrintModal.js

```diff
@@ -52,10 +53,18 @@ class PrintModal extends React.PureComponent {
       isWatermarkModalVisible: false,
       watermarkModalOption: null,
       existingWatermarks: null,
+      selectedPrintOption: this.allPages,
```

Comment:
I think my question regarding this PR is about this change. I'm actually not sure if it's a common or good practice to store a React ref on a React state. Can you explain a little bit more about why we are doing this? Specifically, what problem did you find in the previous implementation, and how does the implementation here resolve that problem?

The `Input` component can be a controlled or uncontrolled component depending on the prop usage. If I'm understanding correctly, in this PR we are trying to make the `Input`a controlled component. That is, the PrintModal component has the full control of whether a print option input is checked or not. The `Input` component in the previous implementation is uncontrolled, and we just listen to the onChange event. I guess I'm not clear about the benefits of making them be controlled?